packages/status-react/src/routes/index.tsx

```diff
@@ -50,15 +51,24 @@ export const Community = (props: Props) => {
     options,
   } = props
 
+  // globalStyles()
   useTheme(theme)
 
   return (
     <Router>
       <ProtocolProvider options={{ publicKey, environment }}>
         <AppProvider options={options}>
           <DialogProvider>
-            <GlobalStyle />
-            <Wrapper>
+            {/* <GlobalStyle /> */}
```

Comment:
discussion: why not delete the global styling all together, given the issue with specificity/ordering, and set just `fontFamily` as bare minimum?